datafusion/core/tests/sqllogictests/test_files/timestamps.slt

```diff
@@ -660,6 +660,19 @@ statement error DataFusion error: Error during planning: Timestamp\(Nanosecond,
 SELECT ts1 + ts2
 FROM foo;
 
+# Timestamp - Timestamp
+query ?
+SELECT '2000-01-01T00:00:00'::timestamp - '2000-01-01T00:00:00'::timestamp;
+----
+0 years 0 mons 0 days 0 hours 0 mins 0.000000000 secs
+
+# large timestamp - small timestamp
+# Todo: Does it need to be fixed for negative values?.
```

Comment:
I think negative values is correct and a valid interval. The intervals are signed according to the arrow spec:

https://github.com/apache/arrow/blob/776ea3f4f3d665f38fc370d0f284760eafa7de45/format/Schema.fbs#L363-L384